styles/globals.css

```diff
@@ -341,3 +341,13 @@ abbr[title] {
 .meeting-card:hover > div:nth-child(2) > p  {
   color: white;
 }
+
+.explorer-menu-wrapper > div > div{
+  border: 1px solid rgb(228, 228, 228);
+  border-radius: 10px;
+  padding: 10px;
+}
+
+.explorer-menu-wrapper > div > div > div > button {
+  margin-top: 0px;
+}
```

Comment:
Can't we add a `className` or span tag there instead of targeting so many nested div elements?

Reply:
The reason I used nested div was that I want to tailor the change only to affect the menu in the Explorer view